Formula/gdk-pixbuf.rb

```diff
@@ -80,7 +80,7 @@ def caveats; <<-EOS.undent
     If you need to manually update the query loader cache, set these variables then run
       #{bin}/gdk-pixbuf-query-loaders --update-cache
     EOS
-  end if build.with?("relocations")
+  end if build.with?("relocations") || HOMEBREW_PREFIX.to_s != "/usr/local"
```

Comment:
I believe that line 83 is meant to suppress defining the `caveats` function at all if the `build.with?` conditional is not met, whereas line 69 is meant to suppress the actual ruby code between lines 69 and 72.